extend_all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+. util.sh
+
+if [[ $# -ne 2 ]] || [[ ! -e "$1" ]] || [[ ! -e "$2" ]] ; then
```

Comment:
`! -e` is `-z`

Reply:
`! -n` is `-z`, `-e` checks if it is a file. I didn't go with `-f` because I thought about using symlinks in the build scripts, now I'm not so sure about that.